web/src/lib/components/admin-page/settings/setting-select.svelte

```diff
@@ -10,12 +10,14 @@
   export let isEdited = false;
   export let number = false;
   export let disabled = false;
+  export let onChange: () => void = () => {};
```

Comment:
This should use `dispatch`. See [setting-switch](https://github.com/immich-app/immich/blob/main/web/src/lib/components/admin-page/settings/setting-switch.svelte) for an example of that.

Reply:
Done, thank you